ONSdigital/census-rh-ui
# Cr 1457 - Update CE Manager Text in NI Pages

# Motivation and Context

Different text is required in the page that a CE Manager in NI will see if they request a new UAC:

http://localhost:9092/ni/requests/access-code/ce-manager/

The same changes are also required in the page that a CE Manager in NI will see if they request a paper form:

http://localhost:9092/ni/requests/paper-form/ce-manager/

And similar changes are also required in this page:

http://localhost:9092/ni/start/code-for-ce-manager/     

# What has changed




- Instead of "You need to visit the Communal Establishment Portal"

the title is now "Communal Establishment Manager Portal"

- Instead of "Visit the Communal Establishment Portal website..."

it is now "Visit the Communal Establishment Manager Portal..."

- Instead of "Alternatively, you can contact us for help."

it is now "Alternatively, you can email census-ce@nisra.gov.uk for help."

# How to test?

Run branch of RHUI locally
Go to http://localhost:9092/ni/start/
Click "request a new access code" link
Enter postcode of an NI CE e.g. BT16 1UN
Select address of NI CE e.g. Aaron House Care Home, 40 Rosneath Gardens, Dundonald, BT16 1UN
Confirm address is correct
Select "Manager" bullet point
See page for http://localhost:9092/ni/requests/access-code/ce-manager/
Also see pages for http://localhost:9092/ni/requests/paper-form/ce-manager/ and http://localhost:9092/ni/start/code-for-ce-manager/ 
Check that the html content looks correct in all three pages
